projects/components/src/header/page/page-header.component.ts

```diff
@@ -122,7 +131,7 @@ interface PageHeaderPreferences {
   selectedTabPath?: string;
 }
 
-export const enum PageHeaderContentAlignment {
-  Column = 'column-alignment',
-  Row = 'row-alignment'
+export const enum PageHeaderContent {
+  Primary = 'page-header-primary-content',
```

Comment:
What is primary and what is secondary?

Reply:
could you please share an example?

Reply:
Primary is projected content for the first row of the header, secondary is for the second row. For example the page `ThreatActorsComponent` requires the content to be projected onto the second row, whereas the page `VulnerabilitiesComponent` requires first row.